### PR TITLE
DNS And Common Name for CES IPs Change

### PR DESCRIPTION
**Pull request by jayeshh123**
This PR is related to the DNS and common name for all CES ips change

### CODE REVIEW

**Comment by digvijay-ukirde** [comment]
@sasikeda Above PR is an enhancement that can act as a load balancer to access the CES IP without actual NLB/ALB. By addition of multiple A records for single ces hostname we are trying to distribute the workload across available reserved CES IPS. Can you please review and let us know if there are any open questions. Thanks.

File: resources/ibmcloud/network/protocol_reserved_ip/protocol_reserved_ip.tf [hunk 1]
```diff
@@ -49,6 +49,23 @@ resource "ibm_dns_resource_record" "a_itself" {
   depends_on  = [ibm_is_subnet_reserved_ip.itself]
 }
 
+resource "ibm_dns_resource_record" "b_itself" {
```
Comment (sasikeda) [inline resources/ibmcloud/network/protocol_reserved_ip/protocol_reserved_ip.tf:52]: Sorry, maybe i did not fully understand it. Here, a DNS record will be created for protocol node IP, and for CES IP addresses, another/separate A record will be created. Is this understanding correct ?

Comment (digvijay-ukirde) [reply resources/ibmcloud/network/protocol_reserved_ip/protocol_reserved_ip.tf:52]: @sasikeda Correct. Here is a sample screenshot (ignore the TTL)
Screenshot 2024-04-04 at 10 39 39 AM

```
[vpcuser@jay-bm-12-client-001 ~]$ ping jay-bm-12-ces-res.cesscale.com
PING jay-bm-12-ces-res.cesscale.com (10.241.17.4) 56(84) bytes of data.
64 bytes from jay-bm-12-ces-2-res.cesscale.com (10.241.17.4): icmp_seq=1 ttl=64 time=0.376 ms
64 bytes from jay-bm-12-ces-2-res.cesscale.com (10.241.17.4): icmp_seq=2 ttl=64 time=0.424 ms

[vpcuser@jay-bm-12-client-001 ~]$ ping jay-bm-12-ces-res.cesscale.com
PING jay-bm-12-ces-res.cesscale.com (10.241.17.6) 56(84) bytes of data.
64 bytes from jay-bm-12-ces-3-res.cesscale.com (10.241.17.6): icmp_seq=1 ttl=64 time=0.562 ms
64 bytes from jay-bm-12-ces-3-res.cesscale.com (10.241.17.6): icmp_seq=2 ttl=64 time=0.599 ms

[vpcuser@jay-bm-12-client-001 ~]$ ping jay-bm-12-ces-res.cesscale.com
PING jay-bm-12-ces-res.cesscale.com (10.241.17.5) 56(84) bytes of data.
64 bytes from jay-bm-12-ces-1-res.cesscale.com (10.241.17.5): icmp_seq=1 ttl=64 time=0.569 ms
64 bytes from jay-bm-12-ces-1-res.cesscale.com (10.241.17.5): icmp_seq=2 ttl=64 time=0.554 ms
```

Comment (sasikeda) [reply resources/ibmcloud/network/protocol_reserved_ip/protocol_reserved_ip.tf:52]: Thank you.

**Pull request merged**